manager/garden_manager.py

```diff
@@ -0,0 +1,68 @@
+"""import abstract class"""
+from models.garden import Garden
+
+
+class GardenManager:
+    """
+    Class representing a Garden Manager.
+
+    Attributes:
+        gardens (list): A list of Garden objects.
+
+    Methods:
+        add_garden(garden)
+            Adds a Garden object to the manager.
+        add_gardens(gardens)
+            Adds multiple Garden objects to the manager.
+        find_all_with_vegetable_garden()
+            Finds all gardens with a vegetable garden.
+        find_all_with_orchard()
+            Finds all gardens with an orchard.
+    """
+
+    def __init__(self, gardens: Garden = None) -> None:
```

Comment:
assuming from naming, I think you meant to pass list of gardens, but why did you type it as a singular garden?